src/Ocelot/Cache/Middleware/OutputCacheMiddleware.cs

```diff
@@ -42,7 +42,7 @@ public async Task Invoke(HttpContext httpContext)
             var downstreamRequest = httpContext.Items.DownstreamRequest();
 
             var downstreamUrlKey = $"{downstreamRequest.Method}-{downstreamRequest.OriginalString}";
-            var downStreamRequestCacheKey = _cacheGenerator.GenerateRequestCacheKey(downstreamRequest);
+            string downStreamRequestCacheKey = _cacheGenerator.GenerateRequestCacheKey(downstreamRequest, downstreamRoute);
```

Comment:
Why is there mixed naming with downstream and downStream?

Reply:
@wast 
Thanks for the review!
I agree it should be `downstreamRequestCacheKey`

Another convention issue is `string downstreamRequestCacheKey`, but it should be `var downstreamRequestCacheKey`.
This is my fault. Maybe during resolving merge conflicts I've skipped the issue.